scriptmodules/ports/abuse.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+# 
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="abuse"
+rp_module_desc="Abuse"
+rp_module_license="GPL https://raw.githubusercontent.com/Xenoveritas/abuse/master/COPYING"
+rp_module_section="exp"
+
+# abuse-lib & abuse-sfx will pull in the older abuse package which only works under X
+# no children or animals harmed though, so whatever..
```

Comment:
Please remove these comments and I will merge. thanks.

Reply:
(the comment regarding the packages can be added just above the getDepends, but the other comment can be removed.)